packages/symfony-php-config/src/ValueObjectInliner.php

```diff
@@ -37,7 +37,7 @@ public static function inlineArgumentObject(
 
     /**
      * @param object|object[] $object
-     * @return InlineServiceConfigurator|InlineServiceConfigurator[]
+     * @return InlineServiceConfigurator|\RectorPrefix20210517\Symfony\Component\DependencyInjection\Loader\Configurator\InlineServiceConfigurator[]
```

Comment:
the rector automation keep adding `\RectorPrefix20210517\Symfony\Component\DependencyInjection\Loader\Configurator\InlineServiceConfigurator[]` which will be changed in next commit as prefixed, it needs `@noRector \Rector\TypeDeclaration\Rector\ClassMethod\AddArrayReturnDocTypeRector`

Reply:
This is a bug in Retcor. It should fixed there too